### PR TITLE
Reorganize event management pages

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Add ingest and usage pages to left nav and reorganize content
- Move images to respective folders
- Change reference links to new pages
- [DOCS-5997](https://datadoghq.atlassian.net/browse/DOCS-5997)

### Merge instructions

- [ ] Please merge after reviewing

[DOCS-5997]: https://datadoghq.atlassian.net/browse/DOCS-5997?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by maycmlee** [comment]
Created [DOCS-6603](https://datadoghq.atlassian.net/browse/DOCS-6603)

File: content/en/service_management/events/_index.md [hunk 1]
```diff
@@ -16,60 +16,34 @@ further_reading:
...
+Join the Event Management beta to correlate Datadog and third party alerts, change events into actionable insights, centralize investigations, and manage critical issues faster as a team. Read the <a href="https://www.datadoghq.com/blog/dash-2022-new-feature-roundup/#event-management">announcement</a> to learn more.
+{{< /callout >}}
 
-Datadog Events gives you a consolidated interface to search, analyze, and filter events from any source in one place.
+## Overview
 
-Without any additional setup, Datadog Events automatically gathers events that are collected by the Agent and installed integrations.
+Ingest, enrich & normalize, and correlate (see public beta) your events from any source into actionable insights. Datadog automatically creates events from various products including monitors, Watchdog, and Error Tracking. You can also track events generated from Agent and installed integrations. Event Management can also ingest events from any sources, including alert events from 3rd parties, change requests, deployments, configuration changes.
```
Comment (drichards-87) [inline content/en/service_management/events/_index.md:34]:
```suggestion
Ingest, enrich and normalize, and correlate (see public beta) your events from any source into actionable insights. Datadog automatically creates events from various products including monitors, Watchdog, and Error Tracking. You can also track events generated from the Agent and installed integrations. Event Management can also ingest events from any sources, including alert events from third parties, change requests, deployments, configuration changes.
```

File: content/en/service_management/events/_index.md [hunk 2]
```diff
@@ -16,60 +16,34 @@ further_reading:
...
-
-{{< img src="service_management/events/generate-metrics.png" alt="Image of metrics with the events search query." >}}
-
-### Normalize and enrich events with processing pipelines
-
-A _processor_ runs data-structuring actions on events attributes when they are ingested. A _pipeline_ is composed of one or multiple processors executed sequentially. With event processing pipelines, you can:
-
-- Normalize disparate sources of events by remapping attributes. For example, use the same reserved [service tags][19] everywhere.
-- Enrich events with external data saved in a [Reference Table][20] (beta). For example, map a service name with your service directory to enrich events with team ownership information, links to dashboards, or links to documentation.
+{{< img src="service_management/events/events_explorer.png" alt="Events explorer display" >}}
```
Comment (drichards-87) [inline content/en/service_management/events/_index.md:38]: I noticed that this screenshot has a Datadog employee email in it. I would recommend retaking this screenshot. If you need to, you can change the email displayed in the UI (and make it something generic) using developer tools > Inspect in the browser. Reach out if you need any help doing that.

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you some light feedback.

File: content/en/service_management/events/_index.md (hunk 2 above)
Comment (estherk15) [reply content/en/service_management/events/_index.md:38]: I reevaluated whether we need another picture of the Events Explorer and removed. Thanks for catching this!

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**